pebble/src/main/java/com/mitchellbosecke/pebble/template/Scope.java

```diff
@@ -96,4 +97,13 @@ public boolean containsKey(String key) {
   public boolean isLocal() {
     return this.local;
   }
+
+  /**
+   * Returns keys of all the variables at this scope.
+   *
+   * @return A set of keys
+   */
+  public Set<String> GetKeys(){
```

Comment:
getKeys

Reply:
Oops! Sorry about that. I have too much C# going on with me.